### PR TITLE
Microverse Project Responsive Web: The Next Web

### PR DESCRIPTION
**Pull request by Janluc**
## The Next Web

- Use of Media Queries for a responsive website

- Use of Flexbox and Grid for the general layout of the website

- Use of semantic HTML for further readability

### CODE REVIEW

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## :wave: :wave: Hi Team!!! Great effort on the project!

:100: Descriptive README
:100: Descriptive enough PULL REQUEST and commit messages.
:100: Using semantic HTML

There are still some issues to be addressed.

# Required Changes
### [Mandatory] :red_circle: 

- [x] nav should be wrapped around the ```<header>``` HTML tag. See comments.

- [x] Assignt alternative text to all images. Don't leave it blank. See inline comments.

- [x] Footer social media elements should be links

- [x] Footer menu elements should be links

#### Mobile

- [x] Extra margin on mobile version
image

- [x] Images not displaying as the original
image

- [x] Articles doesn't have the right amount of margin between them in Latest News and Latest Deals
image

- [x] Text is missing surrounding white space
image

- [x] Missing arrow
image

- [x] Footer menu not resizing properly
image

- [x]  Latest findings not displaying properly
image


#### Tablet (768)

- [x] Navbar is not displaying as original in tablet mode
image


- [x] footer menu not resizing as original
image

#### Desktop (1024)

- [x] Navbar does not match original
image

### [OPTIONAL]  :large_blue_circle: 

- [ ] It would be interesting to try background images using the background CSS property for all images. If you may try this out.

- [ ] Keep the CSS files DRY. See inline comments.

### [Comments] :green_circle: 
- The ```<header>``` element represents a container for introductory content or a **set of navigational links**.

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

File: css/tablet.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+}
+
+.nav-list {
+  display: inline-block;
+}
+
+.nav-hid {
+  display: none;
+}
+
```
Comment (jubaan) [inline css/tablet.css]: - [x] Keep code DRY. Example:
**Actual**
```
.ham-menu {
  display: none;
}

.nav-hid {
  display: none;
}
```
**Improved*** :+1: :
```
.ham-menu,
.nav-hid {
  display: none;
}
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,913 @@
...
+        </div>
+
+        <div class="companies-pad">
+          <div class="companies">
+            <div class="comp-cont">
+              <a href="#">
+                <div class="orange pos-rel">
+                  <img class="company-logo"
+                    src="https://img-cdn.tnwcdn.com/image?url=https%3A%2F%2Fpbs.twimg.com%2Fprofile_images%2F775688252019924992%2F7jJO3joD.jpg&signature=8e3b426e1ed9e113bbde6b7cfeaff335"
+                    alt="">
```
Comment (jubaan) [inline index.html]: - [ ] Fill the ```alt=""``` attribute.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,913 @@
...
+        <li><i class="fas fa-rss"></i></li>
+        <li><i class="fas fa-envelope"></i></li>
+      </ul>
+      <ul>
+        <li>EVENTS</li>
+        <li>ABOUT</li>
+        <li>ADVERTISE</li>
+        <li>JOBS</li>
+        <li>CONTACT</li>
+      </ul>
```
Comment (jubaan) [inline index.html]: - [ ] This elements should be links

**Review by jubaan — CHANGES_REQUESTED** [review]

**Comment by jcy2704** [comment]
@jubaan In the desktop resolution. Do you want the navbar to look like when we are at the top, or do you want it to look like when we scroll a little bit to the bottom?

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required

## Hello team,

**Great job on implementing the requested changes. However, there are still more required changes before the project can be approved.**

## Mandatory

#### 1).  @1024px width

- **Fix the middle navbar so the text doesn't display on multiple lines. (see screenshot below).**

- **Remove the left spacing of the text in the yellow background. (see screenshot below).**

original  |  clone
--------- |  -------
web_original  |  web_clone

#### 2). @1023px width

- **The top images have a bigger height than the original.**

original  |  clone
--------- |  ---------
top_original  |  top_clone

- **The sub-topics below latest funding rounds should display on one line and not multiple lines.**

original | clone
---------| -------
latest_original | latest_clone

3). **The medium breakpoint is at 768px  and not 760px. Please update that.**
breakpoint

#### 4). @759px width

- **Increase the image heights for all sections starting at Hard fork and ending at distract, so they match the original. Here is an example of the Hardfork section.**

original | clone
---------| -------
hardfork_original  | hardfork_clone

 5). **In the latest deals section, the small text below the image captions should be a price, not time.**

original | clone
---------| -------
deals_original |  deals_clone

**Comment by jcy2704** [comment]
Hello, @blackpintz thanks for the code review. However, in point number 2 there seems to be something wrong because on my screen with the same resolution on both websites are almost the same.
**Original**
Original
**Clone**
Clone

**Comment by jubaan** [comment]
> @jubaan In the desktop resolution. Do you want the navbar to look like when we are at the top, or do you want it to look like when we scroll a little bit to the bottom?

I'm just reading this message @jcy2704, I don't like Github notifications a lot of noise and messages like this pass inadvertent. Please next time don't hesitate on reaching me on slack, you can find me like @jubaan 

The idea is to have a navbar as similar to the original in both scenarios at the top and after scrolling., But if it's out of the scope of the programmer, and the project, because maybe doing so requires a technology that is not required like JS, it should match the navbar look and style at the start position.

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

Great work working on the previously required changes.

- [ ]  Please update the live preview link in the README file. The live preview still shows the version without changes.

### [COMMENTS] :green_circle: 
- The README besides the project is one of the most important files of the project. It gives the reader a summary of the programmer's intent. We should maintain the file updated to reflect the latest changes, doing so helps to avoid confusion and misleading.

**Review by jubaan — APPROVED** [review]
# Status: Approved :tada: :tada: :tada:

## :clap: :clap: :clap: Awesome team! Congrats on finishing another project! 

You may now merge your project to master!

## See you at the top :rocket: !!

**Pull request merged**